node/api/renter.go

```diff
@@ -198,6 +198,9 @@ func (api *API) renterHandlerPOST(w http.ResponseWriter, req *http.Request, _ ht
 		} else {
 			settings.Allowance.Hosts = hosts
 		}
+	} else if settings.Allowance.Hosts == 0 {
+		// Sane defaults if host haven't been set before.
+		settings.Allowance.Hosts = recommendedHosts
```

Comment:
hmm are we sure that we want this to be an api thing? I feel like it should be more the client's responsibility to set these defaults.

I guess I'm split on this

Reply:
The big reason being that different clients are going to have different defaults. If you are running a home app, 50 is pretty good. If you are running an SMB cold storage solution, you might want 100 or 200 hosts instead.

Reply:
I'm not a big fan of this either. Unfortunately that was the behavior in the past and I had to add it again to not break compatibility.